api/_mixins/DocumentOrShadowRoot__Document.json

```diff
@@ -295,15 +249,15 @@
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
         }
       },
       "fullscreenElement": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentOrShadowRoot/fullscreenElement",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/fullscreenElement",
```

Comment:
The Opera and Samsung Internet data needs to be mirrored here, they currently says only the prefixed variant is supported.